drsclient/client.py

```diff
@@ -296,15 +296,15 @@ async def _get(self, client_cls, *path, **kwargs):
             resp = await client.get(self.url_for(*path), **kwargs)
             return resp
 
-    @timeout_wrapper
+    @retry_and_timeout_wrapper
```

Comment:
i think you can delete the `timeout_wrapper` function

Reply:
Donezo